aleph-client/src/lib.rs

```diff
@@ -301,3 +303,49 @@ pub fn get_storage_key(pallet: &str, call: &str) -> String {
     let storage_key = StorageKey(bytes.into());
     hex::encode(storage_key.0)
 }
+
+pub type RewardPoint = u32;
+
+/// Helper to decode reward points for an era without the need to fill in a generic parameter.
+/// Reward points of an era. Used to split era total payout between validators.
+///
+/// This points will be used to reward validators and their respective nominators.
+#[derive(Clone, Decode, Default)]
+pub struct EraRewardPoints {
+    /// Total number of points. Equals the sum of reward points for each validator.
+    pub total: RewardPoint,
+    /// The reward points earned by a given validator.
+    pub individual: BTreeMap<AccountId, RewardPoint>,
+}
+
+pub fn get_era_reward_points<C: AnyConnection>(
```

Comment:
Wonder if the better place for this function and below added is file `src/staking.rs`?